### ISSUE TITLE
Bug in `conda_install`

### ISSUE THREAD

**Issue by DataTerminatorX** [rddill-IBM/BlueMix-R-Buildpack#3]
I find that if one is running for the first time, then line42 `conda-env remove -n r_env` in `BlueMix-R-Buildpack/bin/conda_install` should be commented, or it will report error `CondaEnvironmentNotFoundError: Could not find environment: r_env`. I think this is because if this is the first time you build pack, there's no existing `r_env`.)

**Comment by rddill-IBM** [comment]
You are correct in your assessment. I'll research ways to test for the presence of r_env to remove this error.
